runtime/inspector/mod.rs

```diff
@@ -481,58 +117,40 @@ impl DenoInspector {
 
   pub fn new(
     js_runtime: &mut deno_core::JsRuntime,
-    server: Option<Arc<InspectorServer>>,
+    new_websocket_rx: mpsc::UnboundedReceiver<WebSocketProxy>,
```

Comment:
Next step is to rewrite this function to not depend on `WebSocketProxy` - turns out inspector can be agnostic of the "transport" layer - all it cares is to get some kind of channel to pump messages.